### DISCUSSION TITLE
Coordinate system

### DISCUSSION THREAD

**Discussion by theryokato** [SimDynamX/SpaceTeamsPRO#78] [2024 Lunar SAR Competition Q&A] [answered]
I understand what XYZ stands for North, West, and Up. In order to understand the coordinate system better (with my understanding the crashed sites in JSON dont fit within the bounds), are you able to give the name of or a link to this system used by JPL?

**Comment by connorjak** [comment, marked as answer]
The XY coordinates we use for the MoveToCoord are North and West aligned coordinates which project back onto the surface. The Locations in the sim config are PCPF (planet-centered, planet-fixed) coordinates, which are X+: out of 0,0 lat long, Y+ out of 0,90 lat long, Z+ out of the north pole. These are the frame used by the Location functions in the `spaceteams.PlanetUtils.Coord` class.

We will give y'all a script you can use to change out which crash site is used and which obstacle distribution is used soon, probably today.

**Reply by connorjak** [reply]
BTW the reference radius of the moon we use everywhere is 1737400m
